src/components/item/sharing/ConfirmMembership.tsx

```diff
@@ -0,0 +1,92 @@
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+import DialogTitle from '@mui/material/DialogTitle';
+
+import { PermissionLevel } from '@graasp/sdk';
+import { Button } from '@graasp/ui';
+
+import { useCurrentUserContext } from '@/components/context/CurrentUserContext';
+
+import { useBuilderTranslation } from '../../../config/i18n';
+import { mutations } from '../../../config/queryClient';
+import { CONFIRM_MEMBERSHIP_DELETE_BUTTON_ID } from '../../../config/selectors';
+import { BUILDER } from '../../../langs/constants';
+import CancelButton from '../../common/CancelButton';
+
+const labelId = 'alert-dialog-title';
+const descriptionId = 'alert-dialog-description';
+
+type Props = {
+  open?: boolean;
+  handleClose: () => void;
+  item: any;
+  memberToDelete: any;
+  isOneAdmin: boolean;
```

Comment:
```suggestion
  hasOnlyOneAdmin: boolean;
```
Let me know if you don't like this name.